server/.env.example

```diff
@@ -0,0 +1,2 @@
+OBJECT_STORE_BUCKET=gpdqha
```

Comment:
Should we remove the bucket from the example env?

Reply:
Good idea. Also - I think I should make the bucket part of the url parameters.